services/player/api/serializers.py

```diff
@@ -13,15 +13,23 @@ class Meta:
         model = Player
         fields = ('username',)
 
+
 class FirstNameSerializer(serializers.ModelSerializer):
     class Meta:
         model = Player
         fields = ('first_name',)
-        
+
+
```

Comment:
unnecessary new line

Reply:
**Two** new lines between classes, and **One** new line between functions.

Reply:
Which means Two new lines is correct